text/src/autogluon/text/automm/predictor.py

```diff
@@ -949,9 +972,14 @@ def load(
             config=config,
             num_classes=assets["output_shape"],
             num_numerical_columns=len(df_preprocessor.numerical_feature_names),
-            num_categories=df_preprocessor.categorical_num_categories
+            num_categories=df_preprocessor.categorical_num_categories,
+            pretrained=config.pretrained
```

Comment:
I think we can just set pretrained=False, since we are loading from a trained checkpoint saved by AutoMM.

Reply:
+1. @yiqings need to fix it.

Reply:
We follow the original timm settings by setting pretained =True for downloading online pretrained weights, c.f. [https://github.com/rwightman/pytorch-image-models/blob/7c67d6aca992f039eece0af5f7c29a43d48c00e4/timm/models/factory.py#L37](url). Here is the annotations for our `create_model` [https://github.com/yiqings/autogluon/blob/8985d83bf238432ebba5062655a7a4b45bea44d3/text/src/autogluon/text/automm/utils.py#L427](url), in which we call TimmAutoModelForImagePrediction in ./text/src/autogluon/text/automm/models/timm_image.py, and its annotation reads [https://github.com/yiqings/autogluon/blob/8985d83bf238432ebba5062655a7a4b45bea44d3/text/src/autogluon/text/automm/models/timm_image.py#L47](url).

Reply:
Conclusively, by keeping aligned with timm, we write pretained =True for online downloading pretrained weights, and pretained =False for using the local pretrained weights. Probably it does cause same confusion, we may change the name of this flag.

Reply:
Is it because timm models weights are already saved in model.ckpt file?

Reply:
Yes, that's the weights we load in `.load()`. Timm supports not downloading checkpoints if `pretrained=False`.